src/AI/InfantrySpamAI.java

```diff
@@ -126,6 +126,7 @@ public GameAction getNextAction(GameMap gameMap)
 
       // If no attack/capture actions are available now, just move towards a non-allied building.
       Utils.sortLocationsByDistance( new XYCoord(unit.x, unit.y), unownedProperties);
+      //Utils.sortLocationsByTravelTime(unit, unownedProperties, gameMap);
```

Comment:
Adding commented code?

Reply:
Eh, I thought about doing that instead, since it's more correct, and it's possible to construct a scenario where `sortLocationsByDistance` will end up ping-ponging a unit back and forth each turn without getting anywhere.
Sorting by actual travel time is pretty expensive though; you get a noticeable slowdown in Europe, so I switched back. I should remove the comment though.